src/featureFlags/index.js

```diff
@@ -1,6 +1,17 @@
+function stringToBoolean(string) {
+  if (string) {
+    switch (string.toLowerCase().trim()) {
+      case 'true': case 'yes': case '1': return true;
+      case 'false': case 'no': case '0': case null: return false;
+      default: return Boolean(string);
+    }
+  }
+  else return false;
+}
+
 const flags = {
```

Comment:
I could be wrong about this, but I don't see this being imported to anywhere in your tests, or anywhere in any of the Jest config files. Is this variable actually used? If it is, I think it would make more sense to see if we can move it to the test dir because the flags var is only used in testing.

Reply:
The flags var is being used in the mixin of PhotoDetailPage for both client as well as server

Reply:
Ah I see. I think it would be good to add a comment above the var declaration explaining that it's used for mocking the flags for testing.

I'm approving with the expectation that the comment will be added :)

Reply:
Can you move this to a separate module, maybe inside the `utils/` dir? And add the appropriate tests to it, since you're adding tests :)